Makefile

```diff
@@ -283,14 +285,18 @@ web: web-pre-process $(HUGO)
 	# TODO(bwplotka): Make it --gc
 	@cd $(WEB_DIR) && HUGO_ENV=production $(HUGO) --config hugo.yaml --minify -v -b $(WEBSITE_BASE_URL)
 
+.PHONY: failint
+failint:
+	@echo ">> verifying packages being imported"
+	@faillint -paths $(IMPORTS_TO_AVOID) ./...
```

Comment:
Also let's install `failint` as we do for other Go binaries.

Reply:
I've followed the "Dependency management" section under the contributing guide. I've realized that if I run `make deps` it removes the `faillint` entries from `go.sum` and `go.mod`

Reply:
And that's ok - check the binaries like `$(MINIO_SERVER)` and how it install it and version. (: Can we mimic that?

Reply:
Done (but check it, please)!